sys-config/ltoize/files/package.cflags/ltoworkarounds.conf

```diff
@@ -84,6 +83,61 @@ dev-tex/chktex "use pcre && FlagSubAllFlags -flto*" # Segmentation faults as lib
 www-apps/hugo *FLAGS-=-flto*
 sys-libs/musl *FLAGS-=-flto*
 sys-apps/fakechroot *FLAGS-=-flto* # "Cgraph edge statement index out of range" error when linking with LTO enabled
+dev-libs/libltdl *FLAGS-=-flto* # dev-libs/xmlsec fails to build if LTO is enabled
+sys-libs/libapparmor *FLAGS-=-flto* # sys-apps/apparmor fails to build if LTO is enabled
+app-misc/tracker *FLAGS-=-flto* # segfaults in libtracker-sparql-2.0.so.0
+app-crypt/veracrypt *FLAGS-="-flto*" # fails to link as of GCC 9.1
+app-office/libreoffice *FLAGS-="-flto*" # fails as of GCC 9.1. internal compiler error: in record_target_from_binfo, at ipa-devirt.c:2667
+mail-client/thunderbird *FLAGS-="-flto*" # fails to link as of GCC 9.1. <artificial>:(.text._ZN7sandbox3Die9ExitGroupEv+0x5f): undefined reference to `SyscallAsm'
+dev-db/firebird *FLAGS-="-flto*" # gbak segfaults at libEngine12.so during build
+
+# BEGIN: Building with LTO fails due to failed variable substitution
+# See issue #38
+app-crypt/gcr *FLAGS-=-flto*
+app-crypt/libsecret *FLAGS-=-flto*
+app-crypt/p11-kit *FLAGS-=-flto*
+app-i18n/ibus *FLAGS-=-flto*
+app-misc/jq *FLAGS-=-flto*
+app-text/enchant *FLAGS-=-flto*
+app-text/evince *FLAGS-=-flto*
+dev-libs/folks *FLAGS-=-flto*
+dev-libs/libatasmart *FLAGS-=-flto*
+dev-libs/libmaxminddb *FLAGS-=-flto*
+dev-libs/libpeas *FLAGS-=-flto*
+dev-python/dbus-python *FLAGS-=-flto*
+dev-python/notify-python *FLAGS-=-flto*
+gnome-base/gconf *FLAGS-=-flto*
```

Comment:
Unfortunately I can't test these as I can't install Gnome due to not using systemd

Reply:
OK -- I actually just swtiched to systemd just so I could test these last things out.

Reply:
Sorry you had to do that. Only reason I'm using systemd as well is because GNOME is now tightly integrated. I'd rather not use it...

Reply:
Hey it's no problem -- I really ought to learn systemd anyway.  And it's a great opportunity to see where GNOME is at these days.

Reply:
OK -- Got `gnome-base/gnome` fully emerged and it seems to be working here.